.ci/ci_cache_kernel.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Comment:
I thought our general idiom for this didn't have the whole cd/pwd in it - maybe you can use a `readlink -f` like you do further down? This with the cd/pwd just looks odd ;-)